src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,24 @@
+import { INotification } from '../../domain/types'
+import { SocialNetLinks } from '../SocialNetLinks'
+import './Notifications.scss'
+
+interface INotificationComponent {
+  (notifications: string): JSX.Element
+}
+
+export const Notifications: INotificationComponent = (notifications) => {
```

Comment:
What a type of notifications? And i wouldn't see JSON here. Please add additional service for converting JSON to final format of notofication